### ISSUE TITLE
Error on saving image to cache : CGImageDestinationCreateWithData : invalid capacity (0)

### ISSUE THREAD

**Issue by mrikh** [SDWebImage/SDWebImage#2738]
Platform - iOS 12.3
SDWebImage - 5.0.3
Integration - Cocoapods
Xcode - 10.2.1

I am trying to save an image to the cache manually. So what i do is:

`SDImageCache.shared.store(tempImage, forKey: "hmm", completion: nil)`

Whenever the above line is executed i get the following printed to the console:
```
CGImageDestinationCreateWithData:3088: *** ERROR: CGImageDestinationCreateWithData: invalid capacity (0)
```
And when i try to fetch the image from the cache, it is also being fetched correctly. So basically should i be worried about the above console log?

**Comment by dreampiggy** [comment]
This is a warning from Image/IO, because the `CGImageDestinationCreateWithData` pass the `count` arg with 0. It will map this value to 1 instead. So this does not cause any runtime issue.

I'll provide a quick fix for this warning.

**Comment by dreampiggy** [comment]
Fixed in v5.0.4
